behave/wrapper.sh

```diff
@@ -5,10 +5,18 @@
 export A11Y=${A11Y_ENABLED:-false}
 echo "A11Y is $A11Y"
 
+TAGS=""
+# Run only FALA tagged tests
+if [ "$FALA_TESTS_ONLY" = "true" ]
+then
+  TAGS="--tags=@fala"
+fi
+echo "TAGS is $TAGS"
+
 #
-# execute behave
+# execute behave with flags
```

Comment:
This seems to be just floating and not connected to anything. I think you can remove this comment.